librz/util/str.c

```diff
@@ -753,22 +753,22 @@ RZ_API char *rz_str_newf(const char *fmt, ...) {
 	return p;
 }
 
-// Secure string copy with null terminator (like strlcpy or strscpy but ours
+// Secure string copy with null terminator (like strlcpy or strscpy but ours)
 RZ_API size_t rz_str_ncpy(char *dst, const char *src, size_t n) {
 	rz_return_val_if_fail(dst && src, 0);
-	size_t i;
 
 	// do not do anything if n is 0
 	if (n == 0) {
 		return 0;
 	}
+	#if HAVE_STRLCPY
```

Comment:
You need to add the check in the Meson buildsystem for it.

Reply:
How do I do that?

Reply:
See https://github.com/rizinorg/rizin/blob/dev/meson.build#L357